src/components/comment.jsx

```diff
@@ -3,11 +3,13 @@ import Edit from './edit';
 import Delete from './delete';
 import { v4 as uuidv4 } from 'uuid';
 import Comments from '../pages/Comments';
+import Reply from './reply';
 
 const Comment = (props) => {
   const [replyClicked, setReplyClicked] = useState(false);
   const [replyValue, setReplyValue] = useState('');
   const [comment, setComment] = useState(props.comment);
+  const [replies, setReplies] = useState(props.comment.replies);
```

Comment:
is this needed? the `replies` seems to never be used?

Reply:
setReplies is used though so we'll need to create a separate function for it